smailyforprestashop.php

```diff
@@ -72,6 +72,10 @@ public function install()
             !Configuration::updateValue('SMAILY_ABANDONED_CART_TIME', '') ||
             !Configuration::updateValue('SMAILY_SYNCRONIZE_ADDITIONAL', serialize(array())) ||
             !Configuration::updateValue('SMAILY_CART_SYNCRONIZE_ADDITIONAL', serialize(array())) ||
+            !Configuration::updateValue('SMAILY_RSS_CATEGORY_ID', 'all_products') ||
+            !Configuration::updateValue('SMAILY_RSS_LIMIT', '50') ||
+            !Configuration::updateValue('SMAILY_RSS_ORDER_BY', 'date_upd') ||
+            !Configuration::updateValue('SMAILY_RSS_ORDER_WAY', 'desc') ||
```

Comment:
I suggest you change naming of configuration option `ORDER_BY` and `ORDER_WAY` to `SORT_BY` and `SORT_ORDER` respectively.

Reply:
I was trying to keep it consistent with how Prestashop calls them in `Product::getProducts` function. Changed it